lxd/images.go

```diff
@@ -1115,93 +1115,214 @@ func autoUpdateImagesTask(d *Daemon) (task.Func, task.Schedule) {
 }
 
 func autoUpdateImages(ctx context.Context, d *Daemon) error {
```

Comment:
@monstermunchkin please could you expand the commit message from "Fix auto image updates" to explain what is being fixed, as I confess I'm not sure how to review it without knowing what was broken with it :) thanks

Reply:
@monstermunchkin thanks for this, it really helps when reviewing as that gives a good concise summary of what I should look out for :)